### DISCUSSION TITLE
OpenWrt 运行alist 报 Illegal instruction

### DISCUSSION THREAD

**Discussion by Jiao-Xia-Ke** [AlistGo/alist#1571] [Q&A]
OpenWrt 21.02.2   CPU:MT7621 运行alist-linux-musl-mipsle 报 Illegal instruction,最近几个版本都试过了不行,求解决!

**Comment by dew556** [comment]
那是第三方编译的共享在群里，请到qq群咨询，或联系第三方发布者：qq156917266，作者不一定是发布者。
